### PR TITLE
add except, update changelog

### PR DESCRIPTION
**Pull request by sami-sweng**
## Description
Why was this PR created?

## Development notes
What have you changed, and how has this been tested?

## Checklist

- [x] Read the [contributing](https://github.com/Galileo-Galilei/kedro-mlflow/blob/master/CONTRIBUTING.md) guidelines
- [ ] Open this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Update the documentation to reflect the code changes
- [x] Add a description of this change and add your name to the list of supporting contributions in the [`CHANGELOG.md`](https://github.com/Galileo-Galilei/kedro-mlflow/blob/master/CHANGELOG.md) file. Please respect [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines.
- [ ] Add tests to cover your changes

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

**Comment by sami-sweng** [comment]
@Galileo-Galilei I can't see what's failing in Snyk, no access to the org. Should I request access?

**Comment by Galileo-Galilei** [comment]
No don't worry. It's failing for a while because of mlflow dependencies issues. 
Thanks for the PR, I'll have a look soon!

**Comment by sami-sweng** [comment]
Okay, thank you. I can say that I tested with our internal project and building it with kedro-mlflow from this commit and I don't have any further error. So it looks that was the only issue from our perspective with version 0.18.1.

The only issue I can see is that using 0.18.1 would prevent users from configuring with something others than default. But I'm not sure how to proceed with this and since at least the basic functionality is fixed with this PR, adding config support seems less urgent and could be provided in a separate PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 -   :bug: Make `kedro-mlflow` hook log parameters when the project is configured with the `OmegaConfigLoader` instead of raising an error ([#430](https://github.com/Galileo-Galilei/kedro-mlflow/issues/430))
 
+-   :bug: Fix Error when using kedro 0.18.1 with TemplatedConfigLoader ([#452](https://github.com/Galileo-Galilei/kedro-mlflow/issues/452))
```
Comment (Galileo-Galilei) [inline CHANGELOG.md:11]: 2 minor comments: 
- Can you please move it to the "unrealeased" section and create a "fixed" section under this one? 
-  Change the message to be slightly more explicit:  
":bug: Avoid error when using kedro==0.18.1 with ``TemplatedConfigLoader`` and no ``mlflow.yml`` configuration file ([#452](https://github.com/Galileo-Galilei/kedro-mlflow/issues/452))"


The final result should look like this: 

```markdown
# Changelog

## [Unreleased]

### Fixed
-   :bug: Avoid error when using kedro==0.18.1 with ``TemplatedConfigLoader`` and no ``mlflow.yml`` configuration file ([#452](https://github.com/Galileo-Galilei/kedro-mlflow/issues/452))

## [0.11.9] - 2023-07-23

```

**Review by Galileo-Galilei — COMMENTED** [review]
Almost good to go, all the failing tests do not come from you. I'll merge and realease as soon as the changelog is modified. Thank you very much for the PR and sorry for the delay!

**Comment by Galileo-Galilei** [comment]
(And if I remember well, the ``kedro ==0.18.1`` version is a but buggy, you should upgrade if possible)

File: CHANGELOG.md (hunk 1 above)
Comment (sami-sweng) [reply CHANGELOG.md]: done

**Comment by sami-sweng** [comment]
Thank you for the merge!

Do you plan to release soon? I ask but it's not my goal to add any pressure, it's to know if you plan to add other commits in this release I will host the fix on a private pypi server in the meantime while if you plan to do it in the next few days I'll simply wait on the official version.

Thanks for the great work on this lib!

**Comment by Galileo-Galilei** [comment]
It's on PyPI!

**Comment by sami-sweng** [comment]
Excellent, thanks a lot. Yes I'll also check about updating kedo, thanks for the tip!

**Pull request merged**